File: leetcode/solutions/0476.NumberComplement/findComplement_test.ts
import { assertEquals } from "https://deno.land/std/testing/asserts.ts";
import findComplement from "./findComplement.ts";

Deno.test("0476. Number Complement", () => {
  assertEquals(findComplement(5), 2);
  assertEquals(findComplement(1), 0);
});

File: leetcode/solutions/0476.NumberComplement/findComplement_test.go
package solution

import "testing"

func Test_0476_NumberComplement(t *testing.T) {
	result := findComplement(5)
	expected := 2

	if result != expected {
		t.Errorf("TEST FAILED! got: %#v, want: %#v.", result, expected)
	}

	result = findComplement(1)
	expected = 0

	if result != expected {
		t.Errorf("TEST FAILED! got: %#v, want: %#v.", result, expected)
	}
}
